.dotnet/tests/Samples/Chat/Sample03_FunctionCalling.cs

```diff
@@ -64,11 +62,11 @@ public void Sample03_FunctionCalling()
             ChatClient client = new("gpt-3.5-turbo", Environment.GetEnvironmentVariable("OPENAI_API_KEY"));
 
             #region
-            List<ChatRequestMessage> messages = [
-                new ChatRequestSystemMessage(
+            List<ChatMessage> messages = [
+                new SystemChatMessage(
```

Comment:
Naming nit (can be deferred for later): I think we want to keep the role words and "Message" together given they're a bound/domain concept -- so `ChatSystemMessage` or something "creative" like `SystemMessageForChat`

Reply:
(I'd love if we could just drop "chat" from a lot of these types)

Reply:
I'll add this comment to the APIView.